changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ## Changed
 
+- added documentation links for each provider to the readme.
+- added `doc.md` files to each cloud service provider's package folder to the readme.
+- added a comparison of the conventional method with Bifrost's methods for uploading files to GCS and Pinata to the readme.
```

Comment:
Hi @Marzz29 
Great work on this. 

For this part, however, I think these changes are new, they never existed before and should, probably be classified under the `## New` section.

It's thoughful that you kept it under the same version `v0.0.6`.

Reply:
Thank you for your response. I am currently working on implementing the changes.